README.md

```diff
@@ -17,20 +17,20 @@ cloud, and high performance computing (HPC) environments.  CWL is designed to
 meet the needs of data-intensive science, such as Bioinformatics, Medical
 Imaging, Astronomy, Physics, and Chemistry.
 
-<a href="https://open-stand.org/about-us/principles"><img align="left" src="http://standards.ieee.org/images/openstand/128x128-blue2.png" alt="Open Stand badge"></a>
+<a href="https://open-stand.org/about-us/principles"><img align="left" src="https://standards.ieee.org/images/openstand/128x128-blue2.png" alt="Open Stand badge"></a>
 CWL is developed by an informal, multi-vendor working group consisting of
 organizations and individuals aiming to enable scientists to share data
 analysis workflows.  [The CWL project is on Github](https://github.com/common-workflow-language/common-workflow-language) and we follow the [Open-Stand.org principles for collaborative open standards development](https://open-stand.org/about-us/principles/)
 
 
 
-CWL builds on technologies such as [JSON-LD](http://json-ld.org)
-for data modeling and [Docker](http://docker.com) for portable runtime
+CWL builds on technologies such as [JSON-LD](https://json-ld.org)
+for data modeling and [Docker](https://www.docker.com/) for portable runtime
 environments.
 
 ## User Guide
 
-The [CWL user guide](http://www.commonwl.org/user_guide) provides a
+The [CWL user guide](https://www.commonwl.org/user_guide/) provides a
```

Comment:
Oh, yay! I didn't know this had launched.

Reply:
I must admit I don't even understand how it works..

Reply:
Looks like they are using https://letsencrypt.org/ https://www.sslshopper.com/ssl-checker.html#hostname=http://www.commonwl.org/